lib/funky/video.rb

```diff
@@ -40,6 +40,11 @@ def picture
       data[:picture]
     end
 
+    # @return [String] the name of Facebook page for the video.
+    def page_name
+      data.fetch(:from)[:name]
```

Comment:
If we assume they are always there, I suggest

```ruby
def page_name
  data[:from][:name]
end
```

The day this breaks for a specific video, we will switch to `fetch` or `dig`, write the corresponding test and bump the version.

For now, we don't know if/when this will break, so we're better off not pre-optimizing for a _possible_ error that we have never encountered.